front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -45,7 +43,8 @@ const StdcmConfig = ({
   const { updateGridMarginAfter, updateGridMarginBefore, updateStdcmStandardAllowance } =
     useOsrdConfActions() as StdcmConfSliceActions;
 
-  const { pathfindingState } = usePathfindingV2(undefined, pathProperties);
+  const [pathProperties, setPathProperties] = useState<ManageTrainSchedulePathProperties>();
+  const { pathfindingState } = usePathfindingV2(setPathProperties, pathProperties);
```

Comment:
Hm, I think this state is too high up: the path properties may be different for each simulation result. Maybe there are 3 simulation results and each have their own different origin/destination.

(Of course, in this branch there is only a single simulation result, but I think it'd make sense to put the state at the correct level from the start.)